plutus-pab/plutus-pab.cabal

```diff
@@ -39,6 +39,7 @@ common lang
 
 library
     import: lang
+    build-tool-depends: cardano-node:cardano-node, cardano-cli:cardano-cli
```

Comment:
Shouldn't this be on the _tests_? I'm not sure that putting it as a build dependency on the library will do what you want. If you have it as a build tool dependency of tests, I think that you should have them in scope and you shouldn't need the `cabal exec` in the scripts.

Reply:
I moved it to `plutus-pab-test-light`, but I still need to call it using `cabal exec`.

Reply:
Right now I just want to start everything manually. Have access to those binaries in a nix-shell.

Reply:
Ah I see. Yeah, using `cabal exec` for that is a nice solution.

Reply:
Then you maybe don't even need this dependency, but it can't hurt.

Reply:
Tried without and it cannot `cabal exec cardano-cli`.